src/common/keyboard.js

```diff
@@ -92,7 +92,7 @@ if (!('key' in window.KeyboardEvent.prototype)) {
   ( window.KeyboardEvent.prototype
   , 'key'
   , { get: getKey
-    /*::, value: void(0)*/
+    , value: void(0)
```

Comment:
I don't seem to recall a reason properly, but I think only reason `value` was here is because flow was incorrectly expecting `value` even though getter was provided. I don't think uncommenting changes anything, but I'd rather avoid it in case there is something I'm missing to account for.